common/src/main/java/com/alibaba/nacos/common/remote/client/RpcClient.java

```diff
@@ -469,7 +469,7 @@ private boolean healthCheck() {
             return false;
         }
         int reTryTimes = healthCheckRetryTimes;
-        while (reTryTimes > 0) {
+        while (reTryTimes >= 0) {
```

Comment:
I think if it is equal to 0 it will cause one more retry

Reply:
I don't understand what you mean. When `healthCheckRetryTimes` is equal to 0, it means that there will be no retry, only a normal request will be initiated, and no retry will be performed.

Reply:
The old code whether no do request when `healthCheckRetryTimes` is 0?

Reply:
> The old code whether no do request when `healthCheckRetryTimes` is 0?

Yes, in the old code if healthCheckRetryTimes equals 0, no request will be made. The healthCheck method returns false directly.